reference/configuration/framework.rst

```diff
@@ -1145,6 +1146,14 @@ The type of the resource to hint the loaders about the format. This isn't
 needed when you use the default routers with the expected file extensions
 (``.xml``, ``.yaml``, ``.php``).
 
+default_uri
+...........
+
+**type**: ``string``
+
+The default URI used to generate URLs in a non-HTTP context  (see
+:doc:`Generating URLs in Commands </routing>`).
```

Comment:
Hm.. No, this still isn't the correct syntax. 

This will link to the routing page, but not the subsection.

Reply:
I added another comment